index.html

```diff
@@ -62,8 +62,203 @@ <h1>I’m Taflon Glad to see you!</h1>
             </div>
             
         </section>
+        <section class="work_section">
+            <div class="card_container">
+                <div class="card_one">
+                    <div class="card_image">
+                       <img src="images/Snapshoot Portfolio.png" alt="">
+                    </div>
+                    
+                    <h2>Tonic</h2>
+                    <div class="card_details">
+                       <ul>
+                           <li class="canopy">CANOPY</li>
+                            <li>Back End Dev</li>
+                            <li>2015</li>
+                       </ul>
+                      
+                       <p>
+                           A daily selection of privately personalized reads; no accounts or sign-ups required.
+                       </p>
+                    </div>
+                    <div  class="first_buttons">
+                       <a href="#">html</a>
+                       <a href="#">css</a>
+                       <a href="#">Javascript</a>
+                    </div>
+                    <div class="last-button">
+                       <button>See Project</button>
+                    </div>
```

Comment:
- [ ] Dear @Tafloninno, for all your `See Project` buttons we need to follow the button styling: `Active`, `Hover `and `Pressed`
![image](https://user-images.githubusercontent.com/69077061/212305601-6f2d27a2-1c05-4aed-bdd2-17351d0f8f57.png)